master/internal/trials/postgres_trials.go

```diff
@@ -328,13 +328,12 @@ func MetricsTimeSeries(trialID int32, startTime time.Time,
 	timeSeriesFilter *commonv1.PolymorphicFilter, metricType string) (
 	metricMeasurements []db.MetricMeasurements, err error,
 ) {
-	var metricsObjectName, tableName, queryColumn, orderColumn string
+	var tableName, queryColumn, orderColumn string
+	metricsObjectName := model.TrialMetricsJsonPath(model.TrainingMetricType == model.ValidationMetricType)
```

Comment:
this line is evaluating two constants against each other

need `metricType` in comparison?

Reply:
duh! let me see if the tests are failing 🤞🏼 ... yes they are.